### PR TITLE
feat: add UI to add notes and emoji reactions on calls

### PR DESCRIPTION
**Pull request by jamie-rasmussen**
Internal Jira: 
https://wandb.atlassian.net/browse/WB-18700
https://wandb.atlassian.net/browse/WB-18699

Add emoji and note display in calls grid and call details header.

Screenshot 2024-06-24 at 10 25 31 PM

Screenshot 2024-06-24 at 10 26 44 PM

### CODE REVIEW

**Review by nickpenaranda — APPROVED** [review]
LGTM with some small comments about async effects and handling unmount.  i am slightly concerned all the callbacks passed as inline functions will cause a bunch of unnecessary re-rendering, but maybe I missed how we're controlling these re-renders

File: weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/EmojiDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  if (list.length === 0) {
+    return '';
+  }
+  if (list.length === 1) {
+    return list[0];
+  }
+  if (list.length === 2) {
+    return list.join(' and ');
+  }
+  return `${list.slice(0, -1).join(', ')}, and ${list[list.length - 1]}`;
```
[the remark this answers is not in the document]
Comment (jamie-rasmussen) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/EmojiDetails.tsx:35]: I love my children, software engineering and the oxford comma.

File: weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/Reactions.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  readonly = false,
+  forceVisible,
+  twWrapperStyles = {},
+}: ReactionsProps) => {
+  const {loading: loadingUserInfo, userInfo} = useViewerInfo();
+  const [feedback, setFeedback] = useState<Feedback[] | null>(null);
+
+  const parsedRef = parseRef(weaveRef);
+  if (parsedRef.scheme !== 'weave') {
+    throw new Error('Reactions are only valid on weave refs');
```
[the remark this answers is not in the document]
Comment (jamie-rasmussen) [reply weave-js/src/components/PagePanelComponents/Home/Browse3/feedback/Reactions.tsx:31]: We have a bunch of ref code related to artifacts that I think only relates to the "old weave", so we shouldn't encounter it, but type checker complains if we don't handle it somehow.

**Comment by jamie-rasmussen** [comment]
> i am slightly concerned all the callbacks passed as inline functions will cause a bunch of unnecessary re-rendering
 
Fair - I have started a list of some polish items and will add this to it.

**Pull request merged**